performance/models/jump/opf.py

```diff
@@ -38,6 +38,24 @@ def create_model(busfile, branchfile):
         b.q_max /= 100
         b.p_load /= 100
         b.q_load /= 100
+        if not( b.b_shunt_min <= b.b_shunt0 <= b.b_shunt_max ):
+            # The test data sets have invalid data.  As logged warnings
+            # (raised during data validation) significant impact both
+            # the performance test time and time variability, we will
+            # "fix" the data here so that warnings are not generated.
+            #print("DATA warning: b_shunt0[%s] = %s outside (%s, %s)"
```

Comment:
Residual print statement from local testing?

Reply:
No.  Ideally, if someone was to look at this model as an "example", you should print that you are modifying the input data.  I "turned off" that notification because we use the example as a performance test.  It is left in to hint to users what you should consider doing.